src/main/java/code/practice/tasks/Room.java

```diff
@@ -0,0 +1,66 @@
+package code.practice.tasks;
+
+import java.util.Objects;
+
+public class Room {
+
+    private String name;
+
+    private String type;
```

Comment:
I think the good idea here is to use `Enum` with all listed types instead of just `String`.

Reply:
Decided to live as String type for now.